src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -12,9 +12,12 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Roslyn.Test.Utilities
 Imports Roslyn.Test.Utilities.TestBase
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module CompilationUtils
 
+    Private ReadOnly Property _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool()
```

Comment:
>Private ReadOnly Property _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool() [](start = 4, length = 123)

It feels like this can be a readonly field.